spec/rubocop/cop/g2/ban_worker_term_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe RuboCop::Cop::G2::BanWorkerTerm do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for classes that are descendants from Sidekiq::Worker and include the Worker suffix' do
+    expect_offense(<<~RUBY)
+      class FooWorker < Sidekiq::Worker
```

Comment:
`Sidekiq::Worker` is a module so it cannot be subclassed, it needs to be included:
```suggestion
      class FooWorker
        include Sidekiq::Worker
```